examples/lis2hh12/default/main/Kconfig.projbuild

```diff
@@ -0,0 +1,23 @@
+menu "Example configuration"
+    config EXAMPLE_I2C_MASTER_SCL
```

Comment:
Use `rsource "../../../../devtools/Kconfig.i2c` here. The latest examples use it throughout. You just need `EXAMPLE_LIS2HH12_SA0_PIN` in this file.